### PR TITLE
[1LP][RFR] Fix generate report tests

### PR DESCRIPTION
**Pull request by niyazRedhat**
Fixed test_generate_report.py
{{pytest: cfme/tests/intelligence/reports/test_generate_report.py -v --use-provider vsphere55}}

### CODE REVIEW

File: cfme/intelligence/reports/reports.py [hunk 1]
```diff
@@ -539,7 +539,6 @@ def delete(self, cancel=False):
             view.flash.assert_no_error()
         else:
             view.flash.assert_no_error()
-            view.flash.assert_message("Successfully deleted Saved Report from the CFME Database")
```
Comment (quarckster) [inline cfme/intelligence/reports/reports.py]: **REQUIRED**
Why it's deleted?

File: cfme/tests/intelligence/reports/test_generate_report.py [hunk 2]
```diff
@@ -46,4 +46,4 @@ def test_reports_generate_report(request, path):
     """
 
     report = CannedSavedReport.new(path)
-    request.addfinalizer(report.delete_if_exists)
+    request.addfinalizer(report.delete)
```
Comment (quarckster) [inline cfme/tests/intelligence/reports/test_generate_report.py]: **REQUIRED**
If it not exists, exception will be raised.

File: cfme/intelligence/reports/reports.py (hunk 1 above)
Comment (niyazRedhat) [reply cfme/intelligence/reports/reports.py]: Hey when we delete the saved report it doesn't display the flash message. Its bug.I logged that.This is reproducible in 5.7 and 5.8 version. so i think its not required. 
Just have a look ---> https://bugzilla.redhat.com/show_bug.cgi?id=1489387

Comment (quarckster) [reply cfme/intelligence/reports/reports.py]: Don't delete this line, just comment it and add TODO about this bug

Comment (niyazRedhat) [reply cfme/intelligence/reports/reports.py]: Yes right. I will comment.

**Comment by mshriver** [comment]
Tests passing in PRT, results just haven't updated here yet.

**Pull request merged**